app/scripts/lib/util/chrome-util.js

```diff
@@ -35,7 +42,17 @@ module.exports = {
 
   syncGetValue(key) {
     return promised((cb) => {
-      chrome.storage.sync.get(key, (items) => cb(items.hasOwnProperty(key) ? items[key] : undefined));
+      chrome.storage.sync.get(key, (items) => {
+        try {
+          cb(items.hasOwnProperty(key) ? items[key] : undefined);
+        } catch (err) {
+          // Firefox causes an security error when storage.local is accessed from popup scripts.
+          // https://bugzilla.mozilla.org/show_bug.cgi?id=1258139
+          // Perhaps this hack can be removed when storage.sync is officially supported, or the above bug is resolved.
```

Comment:
I think we need this workaround for every other methods (e.g., `localGetValue`) according to [bug 1258139](https://bugzilla.mozilla.org/show_bug.cgi?id=1258139), but currently it seems we are only using `syncGetValue` to get a value from storages, so I put the hack into this method only.